src/main/resources/org/rust/ide/liveTemplates/output.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version='1.0' encoding='UTF-8'?>
+
+<templateSet group="Rust">
+    <template name="p" description="println!" toReformat="true" toShortenFQNames="true"
+              value="println!(&quot;$END$&quot;);">
+        <context>
+            <option name="RUST_STATEMENT" value="true"/>
+        </context>
+    </template>
+
+    <template name="pr" description="print!" toReformat="true" toShortenFQNames="true"
+              value="print!(&quot;$END$&quot;);">
+        <context>
+            <option name="RUST_STATEMENT" value="true"/>
+        </context>
+    </template>
+
+    <template name="pd" description="debug println!" toReformat="true" toShortenFQNames="true"
+              value="println!(&quot;$FORMAT$&quot;, $END$);">
+        <variable name="FORMAT" expression="" defaultValue="&quot;{:?}&quot;" alwaysStopAt="true"/>
```

Comment:
User might want to attach some additional content (which for example I often do). My reasoning is that typing `tab` one more time is much cheaper than having to manually point cursor there